pkgs/tools/compression/gzip/default.nix

```diff
@@ -26,6 +26,12 @@ stdenv.mkDerivation rec {
     }' $out/bin/*
   '';
 
+  # set GZIP env variable to "-n" to stop gzip from adding timestamps
+  # to archive headers: https://github.com/NixOS/nixpkgs/issues/86348
+  setupHook = writeText "setup-hook" ''
```

Comment:
Put `{ stdenv, fetchurl, , writeText, xz }:` in line 1.

Reply:
Fixed in 036e3d0fd57972f164e74fe62af657da237f3705